packages/react/src/components/DataGrid/DataGrid.stories.mdx

```diff
@@ -0,0 +1,73 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import {withDesign} from '../../../.storybook/utils.ts';
```

Comment:
This import is not needed as we haven't included the design for this component

Reply:
Fixed